src/runtime/internal/scheduler.ts

```diff
@@ -33,6 +33,8 @@ export function add_flush_callback(fn) {
 
 let flushing = false;
 const seen_callbacks = new Set();
+let previous_dirty_components = [];
```

Comment:
I suggest to make this a `Set` (like `seen_callbacks` above) instead for performance reasons.

Reply:
Done 👍